plugins/openshift-qe/.claude-plugin/marketplace.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "openshift-qe",
+    "description": "Miscellaenous commands related to testing on OpenShift",
```

Comment:
Not sure if you're in control of this, but this is a uselessly vague description